app/instance-initializers/configure-trackjs.js

```diff
@@ -1,34 +1,52 @@
 import Ember from 'ember';
 
 export function initialize(application) {
-  let trackJs = application.container.lookup('service:trackjs');
-  let appVersion = application.container.lookup('application:main').get('version');
+  const trackJs = application.container.lookup('service:trackjs');
 
   trackJs.configure({
-    version: appVersion
+    version: application.container.lookup('application:main').get('version')
   });
 
-  // http://docs.trackjs.com/Examples/Integrating_with_Ember
-  Ember.onerror = function (err) {
-    if (err.name === 'TransitionError') {
-      Ember.debug('Ignoring TransitionError:', err);
-    } else {
-      trackJs.track(err);
-      Ember.Logger.assert(false, err);
+  Ember.onerror = error => {
```

Comment:
Well, it's shorter, fewer parens, and we don't bind to it, so... Do we need `function` here?